rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json

```diff
@@ -797,23 +797,44 @@
     ]
   },
   "step8": {
-    "title": "Store specimen for courier",
+    "title": "Patient Information",
     "display_back_button": "true",
     "next": "step9",
     "bottom_navigation": "true",
     "bottom_navigation_orientation": "vertical",
     "next_label": "CONTINUE",
     "fields": [
       {
-        "key": "lbl_store_specimen_for_courier",
+        "key": "lbl_patient_confirmation",
         "type": "label",
         "text_color": "#000000",
         "text_size": "10sp",
         "top_margin": "15dp",
         "has_bg": true,
         "bg_color": "#ffffff",
         "label_text_style": "bold",
-        "text": "Store the respiratory sample appropriately until they have been picked up by the courier"
+        "text": "Please confirm the information"
+      },
+      {
+        "key": "patient_detail",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "check_box",
+        "options": [
+          {
```

Comment:
`unique_id` should only show for manual sample details collection i.e. if barcode was not scanned. 

So this could be a separate checkbox with relevance checks.

Reply:
Actually, it's better to break all the options into separate checkboxes since they are not part of a single response.

Reply:
i have made all the checkbox separate and could you more elaborate unique_id section

Reply:
https://github.com/onaio/rdt-standard/pull/456/commits/802098de9d978259cba367a5e781cf4003f15bae#r477089729

Reply:
In the sample collection flow we have two paths - one to scan the barcode and the other to manually record the sample details.

The `unique_id` checkbox should only show if the manual option was selected.

Reply:
There are two paths when collecting a sample - one is to scan a barcode while the other is to enter the sample details manually.

The `unique_id` checkbox should only be shown if manual entry was selected.

Reply:
is there any example work done in somewhere. so that I can take an example of relevance.

Reply:
This is the [relevance file](https://github.com/onaio/rdt-standard/blob/master/rdt-open-reader/src/covidIndonesia/assets/rule/sample-collection-form-relevance-rules.yml) for sample collection form. 

A file can be [marked](https://github.com/onaio/rdt-standard/blob/dc90494ea4ae8ada194408b62fc13390988126d9/rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json#L162) as depending on relevance rules.

Reply:
Extra documentation on relevance can be found in the native forms library documentation.

Reply:
fixed.